pages/api/swagger-to-actions.ts

```diff
@@ -156,26 +156,24 @@ export default async function handler(
         responses: methodObj?.responses ?? null,
       });
     }
-    // Don't insert if already in database (previously uploaded this swagger file)
-    const actionResp = await supabase
-      .from("actions")
-      .select("*")
-      .eq("org_id", orgId)
-      .in(
-        "name",
-        actionInserts.map((action) => action.name)
-      );
-    if (actionResp.error) throw actionResp.error;
-    const existingActionNames = actionResp.data.map((action) => action.name);
-    actionInserts = actionInserts.filter(
-      (action) => !existingActionNames.includes(action.name!)
+  }
+  // Don't insert if already in database (previously uploaded this swagger file)
+  const actionResp = await supabase
+    .from("actions")
+    .select("*")
+    .eq("org_id", orgId)
+    .in(
+      "name",
+      actionInserts.map((action) => action.name)
     );
-    const actionInsertResp = await supabase
-      .from("actions")
-      .insert(actionInserts);
-    if (actionInsertResp.error) {
-      throw actionInsertResp.error;
-    }
+  if (actionResp.error) throw actionResp.error;
```

Comment:
Not necessary now, but is there something smart we can do with the presets (similar to previous dumb comment)? For example having the presets already in the database then copying them to a new org when a new user wants to use them, might that be faster?

Reply:
Hmm nothing easy I can think of! Maybe we could do something like:
1. Always have 1 org for each preset set up but with no users
2. When a new user joins and chooses a preset, we just associate him with the correct preset org
3. We generate a new preset org not associated with anyone yet (to fill the gap so we get back to state 1)

However, what happens if 2 users join in swift succession and there isn't a spare preset org lying around? Or what happens if there's an error when making the new preset? Etc etc - think this opens up a lot of scope for painful bugs